src/ui/components/VPNConnectionStability.qml

```diff
@@ -8,155 +8,124 @@ import Mozilla.VPN 1.0
 import "../themes/themes.js" as Theme
 
 RowLayout {
+    property var numGridColumns: grid.columns
+    onNumGridColumnsChanged: col.handleMultilineText();
```

Comment:
1. Where is this signal documented? Is it part of QT?
2. In QT/QML the connection should be the other way around. The column should connect the rowLayout to its slot. See https://doc.qt.io/qt-5/qtqml-syntax-signals.html

Reply:
Maybe we can see if we can work on these in a separate patch (and maybe in other QML files too)